truss/cli/cli.py

```diff
@@ -881,17 +881,44 @@ def push_training_job(config: Path, remote: Optional[str]):
         BasetenRemote, RemoteFactory.create(remote=remote)
     )
     with loader.import_target(config) as training_project:
-        training_resp = remote_provider.api.upsert_training_project(
+        project_resp = remote_provider.api.upsert_training_project(
             training_project=training_project
         )
 
         prepared_job = deployment.prepare_push(
             remote_provider.api, config, training_project.job
         )
-        remote_provider.api.create_training_job(
-            project_id=training_resp["id"], job=prepared_job
+        job_resp = remote_provider.api.create_training_job(
+            project_id=project_resp["id"], job=prepared_job
         )
 
+        console.print("✨ Training job successfully created!", style="green")
+        console.print(
+            f"🪵 View logs for your job via "
+            f"[cyan]`truss train logs --project-id {project_resp['id']} --job-id {job_resp['id']}`[/cyan]"
+        )
+
+
+@train.command(name="logs")
+@click.option("--remote", type=str, required=False, help="Remote to use")
+@click.option("--project-id", type=str, required=True, help="Project ID.")
+@click.option("--job-id", type=str, required=True, help="Job ID.")
+@log_level_option
+@error_handling
+def get_job_logs(remote: Optional[str], project_id: str, job_id: str):
```

Comment:
Mentioned in the PR description, but this is just the bare minimum to get something usable. Improvements include:

- Better interface, so users don't have to pass back the project / job ID
- Watch interface
- Ability to control limit, time range, etc